packages/amplify-migration-tests/src/migration-helpers/check-version.ts

```diff
@@ -34,4 +34,4 @@ export const validateVersionsForMigrationTest = async (): Promise<void> => {
  *
  * ext migrate flow: https://github.com/aws-amplify/amplify-cli/pull/8806
  */
-export const allowedVersionsToMigrateFrom = ['5.2.0', '6.1.0', '10.5.1'];
+export const allowedVersionsToMigrateFrom = ['5.2.0', '6.1.0', '10.5.1', '11.0.3'];
```

Comment:
I saw `11.0.0` as base on one of other PRs. @danielleadams do we want 11.0.0 or 11.0.3 ? or both?